testlapack/general.go

```diff
@@ -1038,3 +1038,85 @@ func unbalancedSparseGeneral(m, n, stride int, nonzeros int, rnd *rand.Rand) bla
 	}
 	return a
 }
+
+func columnOf(a blas64.General, j int) []float64 {
+	if j < 0 || a.Cols <= j {
+		panic("bad column index")
+	}
+	col := make([]float64, a.Rows)
+	for i := range col {
+		col[i] = a.Data[i*a.Stride+j]
+	}
+	return col
+}
+
+func isRightEigenvectorOf(a blas64.General, xRe, xIm []float64, lambda complex128, tol float64) bool {
```

Comment:
I added a function comment, some comments in the code, and made the code less dense. I think it is better now. Done.